### PR TITLE
Remove commons-lang3

### PR DESCRIPTION
**Pull request by timja**
See: https://github.com/jenkinsci/workflow-cps-global-lib-plugin/pull/93#pullrequestreview-391129809

@jglick @dwnusbaum @bitwiseman

### CODE REVIEW

**Comment by timja** [comment]
Ran a basic smoke test which seemed to work, the library was loaded:
```
library 'pipeline'
buildPlugin()
```

**Comment by dwnusbaum** [comment]
@timja Thanks for filing the PR! I think that normal cases should work fine, my main concern is if @jglick had some particular reason not to use `InvokerHelper`. Maybe it was not used so that Groovy-specific features would not work with the library step to avoid potential sandbox issues?

**Comment by jglick** [comment]
> if @jglick had some particular reason not to use `InvokerHelper`

To the extent I can recall now, I think I just did not realize it was available.

File: src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryStep.java [hunk 1]
```diff
@@ -265,16 +263,10 @@ public AutoCompletionCandidates doAutoCompleteIdentifier(@AncestorInPath ItemGro
         @Override public Object invokeMethod(String name, Object _args) {
             Class<?> c = loadClass(prefix + clazz);
             Object[] args = _args instanceof Object[] ? (Object[]) _args : new Object[] {_args}; // TODO why does Groovy not just pass an Object[] to begin with?!
```
Comment (dwnusbaum) [inline src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryStep.java:265]: nit: `InvokerHelper` handles this conversion internally via `InvokerHelper.asArray`, so I think this line could be removed now.

**Pull request merged**